scenes/src/styles.scss

```
@@ -0,0 +1,32 @@
+@import '~@angular/material/prebuilt-themes/indigo-pink.css';
+
+
+@import '~@angular/material/theming';
+
+@include mat-core();
+
+$primary: mat-palette($mat-red, 500, 200, 900);
+$accent: mat-palette($mat-red, 500, 200, 700);
+$background-color: #fff2f4;
+$secondary-text-color: map_get($mat-red, 600);
+
+$warn: mat-palette($mat-red);
+
+$theme: mat-light-theme($primary, $accent, $warn);
```

```suggestion
$primary: mat-palette($mat-red, 500, 200, 900);
$accent: mat-palette($mat-red, 500, 200, 700);
$warn: mat-palette($mat-red);

$theme: mat-light-theme($primary, $accent, $warn);
$background-color: #fff2f4;
```

And move `$secondary-text-color: map_get($mat-red, 600);` down closer to where it's used (where all the `foreground` stuff is.